### ISSUE TITLE
wxListbook page labels invisible with no images and wxLB_{TOP|BOTTOM}

### ISSUE THREAD

**Issue by PBfordev** [wxWidgets/wxWidgets#24110]
(Based on [the report on wxForum](https://forums.wxwidgets.org/viewtopic.php?f=23&p=221625#p221625), see also [the thread on wx-users](https://groups.google.com/g/wx-users/c/TbJUwrq_iWw/m/XojZ5p-aAQAJ) about the labels size computed only from images).

### Description
When `wxListbook` uses `wxLB_TOP` or `wxLB_BOTTOM`  style and and no images for page labels, the height of page label area (i.e., the `wxListView` managing the labels) is so small, that the page labels are not visible.
![wx-listbook-bug](https://github.com/wxWidgets/wxWidgets/assets/12495521/e15b484c-4ebd-4c0d-9761-5b0d63ad69a4)

#### To Reproduce:
1. Launch the notebook sample.
2. In the menu `File` / `Type` menu switch to `Listbook`.
3. In the menu `File` uncheck `Show Images`.
4. In the menu `File` / `Orientation` switch to `Top`.
5. Observe that the page label area is just few pixels tall.
![wx-listbook-bug](https://github.com/wxWidgets/wxWidgets/assets/12495521/a5699bfc-3b0d-4247-b535-178b25c9d854)

### Platform and version information
- wxWidgets version: GIT master and 3.2.4 (works with 3.0.5)
- wxWidgets port: wxMSW (works on wxGTK)
- OS: Windows 10 v22H2 build 19045.3693 (light mode)

**Comment by basiliscos** [comment]
The issue is still relevant with 3.2.5

**Comment by vadz** [comment]
Yes, it also happens with master. This shouldn't be difficult to fix.
